contributing_to_docs/contributing_user_stories.adoc

```diff
@@ -0,0 +1,69 @@
+[[contributing-user-stories]]
+= Contribute user stories to OpenShift documentation
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+:description: Basic information about how to create user stories for OpenShift GitHub repository
+
+toc::[]
+
+== Modularization back story
+OpenShift docs are getting modularized, starting from OpenShift 4.0. Older content will
+be rewritten from scratch to conform to our modularization guidelines and new feature content
+will be created using user stories.
+
+== How do I contribute user stories?
+Familiarize yourself with the Customer Content Services (CCS)
+link:https://redhat-documentation.github.io/modular-docs/[modularization guide]. This is an important
+first step in making sure you are helping the docs team create content that is fully modularized.
+
+== What is the branch setup for modularized docs?
+
+* link:https://github.com/openshift/openshift-docs/tree/enterprise-4.0[enterprise-4.0]
++
+This builds both OCP 4.0 and Origin/Upstream content. Origin content will be separated using
+ifdef statements in content and distro statements in the _topic_map.yml.
++
+Only CCS submits and merges their PRs here till OCP 4.0 is released, unless others are keen to submit
+content in modularized format.
+
+* https://github.com/openshift/openshift-docs/tree/master[master]
+This builds legacy (existing), mostly non user story based content. This branch will continue to update
+docs.okd.io (upstream) till we switch to 4.0 with the enterprise-4.0 branch.
++
+Please continue to submit your PRs to this branch and specify if it is for 3.11 and/or 4.0.
+If the submitted PR is for the 3.11 branch, CCS merges with follow up edits in 3.11.
+CCS then creates user stories for its inclusion in enterprise-4.0 branch, and if it is completely new content,
+in the enterprise-3.11 branch as well.
+
+* https://github.com/openshift/openshift-docs/tree/enterprise-3.11[enterprise-3.11]
+This branch contains content for the 3.11 branch, and the last branch that contains content that is not completely
+modularized or written from scratch.
++
+This may contain user stories for content for completely new features.
+
+== Do you have a template for submitting user stories if I don't want o contribute content in a modularized format?
+Yes, and this assumes that you are supplying the user story for CCS to create the modularized content from.
```

Comment:
I'd say something like this instead:

```
Instead of a template, we have a series of questions for you to answer to provide us with the information that we need to write the user story and start drafting the content.

The basic format of a user story is:

----
As a <type of user>, I want to <goal state> because <reason behind the goal>.
----

For example, "As a cluster administrator, I want to enable an Auto Scaling group to manage my OpenShift Enterprise cluster deployed on AWS because I want my node count to scale based on application demand."
```